resources/views/layouts/admin-app.blade.php

```diff
@@ -13,10 +13,9 @@
   <!-- FontAwesome -->
   <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.0/css/all.min.css" integrity="sha512-iecdLmaskl7CVkqkXNQ/ZH/XLlvWZOJyj7Yy7tcenmpD1ypASozpmT/E0iPtmFIB46ZmdtAc9eNBvH0H/ZpiBw==" crossorigin="anonymous" referrerpolicy="no-referrer" />
 
-
-  <link rel="stylesheet" href="{{asset('css/button.css')}}">
-  <link rel="stylesheet" href="{{asset('css/form.css')}}">
-  <link rel="stylesheet" href="{{asset('css/admin-style.css')}}">
+  <link href="{{ asset('css/app.css') }}" rel="stylesheet">
```

Comment:
there's no need for the app.css since we don't have `app.scss`

Reply:
I will remove it.